api/api/V1/User.ts

```diff
@@ -257,4 +258,34 @@ export default function (app: Application, baseUrl: string) {
       euaVersion: config.euaVersion,
     });
   });
+
+  app.patch(
+    `${apiUrl}/changePassword`,
+    validateFields([body("token"), body("password")]),
+    fetchUnauthorizedRequiredUserByResetToken(body("token")),
+    validPasswordOf(body("password")),
```

Comment:
This `validPasswordOf(body("password"))` should go where `body("password")` is in the `validateFields` function.